src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const rentalCosts = days * PRICE_PER_DAY;
+
+  if (days < SHORT_TERM) {
+    return rentalCosts;
+  }
+
+  if (days < LONG_TERM && days >= SHORT_TERM) {
+    return rentalCosts - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
```

Comment:
You don't need to add the last condition, as the return in the previous "if" block will terminate the function execution if the condition is true. We don't want to do that becuz if the "day" is incorrect value or it didn't pass the 3 previous conditions - our function will return "undefined" 

[link](https://highload.today/undefined-v-javascript/#3)

so u can just remove it and leave return 
```return rentalCosts - LONG_TERM_DISCOUNT;```